openml_OS/views/pages/frontend/guide/subpage/developers.php

```diff
@@ -24,7 +24,7 @@
         <a href="downloads/ExpDB_SNAPSHOT.sql.gz" class="btn btn-primary"><i class="fa fa-cloud-download fa-lg"></i> Nightly database SNAPSHOT</a><br><br>
 
 	      <p>If you want to work on the website locally, you'll also need the schema for the 'private' database with non-public information.</p>
-        <a href="downloads/openml.sql.gz" class="btn btn-primary"><i class="fa fa-cloud-download fa-lg"></i> Private database schema</a>
+        <a href="downloads/openml.sql" class="btn btn-primary"><i class="fa fa-cloud-download fa-lg"></i> Private database schema</a>
```

Comment:
These pages are not used anymore. The whole of frontend/guide can be removed. An edit such as this one can be made in the new docs by clicking the edit icon on the website: https://docs.openml.org/developers/